Controllers/SensorController.cs

```diff
@@ -9,40 +13,41 @@ public class SensorController
 {
 	private readonly IConfiguration _configuration;
 
+	private readonly OpenRemoteApi openRemoteApi = new OpenRemoteApi();
+
+
 	[HttpGet]
 	[Route("/Sensor")]
-	public List<Sensor> GetSensors()
+	public async Task<List<Sensor>> GetSensors()
 	{
-		return [];
-	}
+		AssetQuery query = new AssetQueryBuilder()
+			.SetLimit(0)
+			.IsRecursive(true)
+			.AddName(new AssetQuery.Name(AssetQuery.NameMatch.CONTAINS, false, "esp32", false))
+			.Build();
 
-	[HttpGet]
-	[Route("/Sensor/Test")]
-	public List<Sensor> GetDummySensors()
-	{
-		var rand = new Random();
+		var queryAssets = await openRemoteApi.QueryAssets(query);
 
-		return
-		[
-			new Sensor
+		var sensors = queryAssets.Select(asset =>
+		{
+
+			// Extracting the coordinates array
+
+			var coordinatesInfo = ((JObject)asset.Attributes["location"].Value).ToObject<CoordinatesInfo>();
+
+
+			//TODO Fix vallue
+			return new Sensor
 			{
-				Id = 1,
-				Name = "Example Sensor 1",
+				Id = asset.Id,
+				Name = asset.Name,
 				RoomId = 1,
-				Value = (float)Math.Round(rand.NextSingle(), 2),
+				Value = 0.5f,
```

Comment:
Sensor value still has static value

Reply:
That will be fixed by @SlipperyTrout with another PR, so for now it's a static value.

Reply:
Will make it a random number for the time being